UnityProject/Assets/Scripts/Messages/Server/ServerMessage.cs

```diff
@@ -17,14 +17,13 @@ public void SendTo(GameObject recipient)
 	{
 		NetworkIdentity netIdentity = recipient.GetComponent<NetworkIdentity>();
 
-		//Only send to the client of the currently owned player as some
-		//netID's being used in this method could be dead players, exclude them:
```

Comment:
The comments I put here were trying to explain that the old way was sending the message to the connection of each dead person as well. Now we just send it to the connection of a player that is in the playerlist instead which makes sure we are only sending the message once to the client. I felt people would lose the meaning of why I made that condition over time